recipes/openmvg/all/conanfile.py

```diff
@@ -66,7 +68,12 @@ def requirements(self):
         self.requires("coin-utils/2.11.6")
         self.requires("eigen/3.4.0", transitive_headers=True)
         self.requires("flann/1.9.2", transitive_headers=True, transitive_libs=True)
-        self.requires("libjpeg/9e")
+        if self.options.with_jpeg == "libjpeg":
+            self.requires("libjpeg/9e")
+        elif self.options.with_jpeg == "libjpeg-turbo":
+            self.requires("libjpeg-turbo/2.1.5")
```

Comment:
```suggestion
            self.requires("libjpeg-turbo/3.0.0")
```
Can probably bump the version soon after #18417 is merged.